imagersite/imagerprofile/templates/home.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block main %}
+  <main>
+    <div class='cover'>
+      {% if photo %}
+        <img src="{{ MEDIA_ROOT }}{{ photo.image.url }}">
+      {% else %}
+       <p>It seems there is nothing here...</p>
```

Comment:
You'll want to replace this somehow with an image that can take the place of a user-uploaded image.